qa/os/src/test/java/org/elasticsearch/packaging/test/ArchiveTestCase.java

```diff
@@ -263,6 +263,64 @@ public void test60AutoCreateKeystore() throws Exception {
         });
     }
 
+    public void test61PasswordProtectedKeystore() throws Exception {
```

Comment:
The packaging tests have been reorganized now, and I think it would be better to have these tests in a new KeystoreToolTests or something like that (you can move the existing keystore tests there as well). Additionally, the intent of the new structure is to allow tests like this to be written as a single test suite that will work for both archives and packages for things that apply to both.

Reply:
That's a great suggestion. And doing this will resolve a couple of your other comments about test coverage. I'll make it happen.